target/linux/ipq806x/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -9,6 +9,16 @@ PHYNBR=${DEVPATH##*/phy}
 . /lib/functions.sh
 . /lib/functions/system.sh
 
+board=$(board_name)
+
+case "$board" in
+	ubnt,unifi-ap-hd)
+			 echo $(macaddr_add $(mtd_get_mac_binary EEPROM 0x6) $(($PHYNBR + 1)) ) > /sys${DEVPATH}/macaddress
+		;;
+    *)
+        ;;
```

Comment:
whitespace from spaces to tabs, like i mentioned

Reply:
Thanks, fixed. My editor tried to be smart...